frontend/src/ui-kit/components/Layout/Header.tsx

```
@@ -0,0 +1,174 @@
+    openSearch();
+  }, [openSearch]);
+
+  // Detect platform for keyboard shortcut hint
+  const isMac = typeof navigator !== 'undefined' && /Mac|iPod|iPhone|iPad/.test(navigator.platform);
+  const shortcutKey = isMac ? 'Cmd' : 'Ctrl';
+
+  return (
+    <header
+      className="sticky top-0 z-30 flex h-16 items-center justify-between border-b border-border-default bg-surface-primary px-4 lg:px-6"
+      role="banner"
+    >
```

Biome (2.1.2)

[error] Interactive elements should not be assigned non-interactive roles.

WAI-ARIA roles should not be used to convert an interactive element to a non-interactive element.
Wrap your interactive element in a <div> with the desired role or put the content inside your interactive element.
Unsafe fix: Remove the role attribute.


(lint/a11y/noInteractiveElementToNoninteractiveRole)